src/rulesets/mergers/__tests__/rules.jest.test.ts

```diff
@@ -211,12 +214,49 @@ describe('Ruleset rules merging', () => {
           ...JSON.parse(JSON.stringify(baseRule)),
           recommended: false,
         },
+        test3,
       },
       'recommended',
     );
 
     expect(rules).toHaveProperty('test.severity', DiagnosticSeverity.Warning);
     expect(rules).toHaveProperty('test2.severity', -1);
+    expect(rules).toHaveProperty('test3.severity', DiagnosticSeverity.Warning);
+  });
+
+  it('rules with no severity and no recommended set are treated as warnings', () => {
```

Comment:
This message makes it sound like there is a crossover between severity and recommended, and im not sure there is?

Reply:
There is some. Basically, if there is no severity set, we either set a default value or disable it. Whether we set a default value or set it to off depends on the value of recommended.

In code, it covers this line https://github.com/stoplightio/spectral/pull/719/files#diff-7ba880b07a41aa1bf9a31d0a3bca8445R116